server/elasticsearch/src/main/resources/hawkular-apm-mapping.json

```diff
@@ -17,11 +17,6 @@
             }],
             "date_detection" : false,
             "numeric_detection": false,
-            "_timestamp": {
```

Comment:
Yes that is fine - but can you create the jira for it and include ref to it here.